packages/rest/src/validation/request-body.validator.ts

```diff
@@ -79,11 +56,23 @@ function convertToJsonSchema(openapiSchema: SchemaObject) {
   return jsonSchema;
 }
 
-// tslint:disable-next-line:no-any
-function validateValueAgainstJsonSchema(body: any, schema: any) {
-  const ajv = new AJV({allErrors: true});
+function validateValueAgainstJsonSchema(
+  // tslint:disable-next-line:no-any
+  body: any,
+  // tslint:disable-next-line:no-any
+  schema: any,
```

Comment:
Maybe we can use `object` instead of `any`? `object` allow only object values, forbids primitives like `number` or `string`, forbids `undefined` and probably `null` too.

Reply:
I changed it to `AnyObject` imported from `@loopback/repository`